src/parser/parser_support.c

```diff
@@ -11669,12 +11830,10 @@ pt_convert_dblink_dml_query (PARSER_CONTEXT * parser, PT_NODE * node, char *sql_
   val->type_enum = PT_TYPE_CHAR;
   val->info.value.string_type = ' ';
 
-  assert (sql_user_text && sql_user_text[0]);
-  if (sql_user_text)
-    {
-      val->info.value.data_value.str = pt_make_remote_query (parser, sql_user_text, snl);
-      PT_NODE_PRINT_VALUE_TO_TEXT (parser, val);
-    }
+  parser->custom_print |= PT_PRINT_SUPPRESS_SERVER_NAME;
```

Comment:
Isn't PT_SUPPRESS_QUOTES also necessary?

Reply:
It's responsiblity of user for the wrong specifying a name.

Reply:
It's not that I used the wrong name.

Both Oracle and CUBRID can use reserved words as identifiers. Oracle uses reserved words by enclosing them in double quotations(" "), and CUBRID uses them by enclosing them in square brackets([ ]), double quotations(" "), and backquotes(` `).

The problem is that the parser_print_tree function only uses square brackets when printing reserved words. An error will occur if the result output by the parser_print_tree function is used as an oracle dblink query.

Reply:
I'm going to handle this issue as a separate issue.